library_generation/test/resources/proto_path_list.txt

```diff
@@ -1,25 +1,17 @@
 # This file is used in integration test against `generate_library.sh`.
 # Format:
 # proto_path repository_path skip_postprocessing_test
-# google/bigtable/admin/v2 java-bigtable true
-# google/bigtable/v2 java-bigtable true
-google/cloud/apigeeconnect/v1 java-apigee-connect false
-google/cloud/asset/v1p5beta1 java-asset false
-# google/cloud/asset/v1p2beta1 java-asset false
-google/cloud/asset/v1p1beta1 java-asset false
-google/cloud/asset/v1p7beta1 java-asset false
-google/cloud/asset/v1 java-asset false
-# google/cloud/dialogflow/v2beta1 java-dialogflow false
-# google/cloud/dialogflow/v2 java-dialogflow false
-# google/cloud/compute/v1 java-compute false
-google/cloud/kms/v1 java-kms false
-google/cloud/redis/v1 java-redis false
-google/cloud/redis/v1beta1 java-redis false
-# google/example/library/v1 google-cloud-example-library-v1-java null false
-google/devtools/containeranalysis/v1 java-containeranalysis false
-google/iam/v1 java-iam true
-google/iam/v2 java-iam false
-google/iam/credentials/v1 java-iamcredentials false
-google/logging/v2 java-logging true
-google/pubsub/v1 java-pubsub true
-google/storage/v2 java-storage true
+google/bigtable/admin/v2,google/bigtable/v2 java-bigtable
+google/cloud/apigeeconnect/v1 google-cloud-java/java-apigee-connect
+google/cloud/asset/v1p5beta1,google/cloud/asset/v1p2beta1,google/cloud/asset/v1p1beta1,google/cloud/asset/v1p7beta1,google/cloud/asset/v1 google-cloud-java/java-asset
```

Comment:
I thought `generate_composed_library.sh` should be used to generate a library that has multiple `proto_path`, e.g., `google/bigtable/v2` and `google/bigtable/admin/v2`.

Is `google/cloud/asset` a good example here?

Reply:
As we discussed in our meeting, the integration test will now rely on `composed_library` regardless of the number of versions, and we will assemble a single "product" (i.e. multiple service versions) with a single call to `composed_library`. So to answer, yes this script basically handles multiple versions (of any service), which solves the case for Java Bigtable and the special `dialogflow` case in google-cloud-java (which requires more than one service version to be available).

Reply:
That said, it is still convenient to add multiple versions of a same service that don't fall into problems with owlbot.py files (i.e. the `asset` example) and run the postprocessor once to ensure better generation times.

Reply:
Makes sense, thanks.